### PR TITLE
Handle coordinator namespace correctly in ConsensusCommitAdmin

### PR DESCRIPTION
**Pull request by brfrn169**
## Description

The coordinator namespace is a reserved namespace in Consensus Commit, requiring special handling. This PR adds specific handling for the coordinator namespace in `ConsensusCommitAdmin`.

## Related issues and/or PRs

N/A

## Changes made

- Added specific handling for the coordinator namespace in `ConsensusCommitAdmin`.

## Checklist

- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation to reflect the changes.
- [x] Any remaining open issues linked to this PR are documented and up-to-date (Jira, GitHub, etc.).
- [x] Tests (unit, integration, etc.) have been added for the changes.
- [x] My changes generate no new warnings.
- [x] Any dependent changes in other PRs have been merged and published.

## Additional notes (optional)

N/A

## Release notes

N/A

### CODE REVIEW

File: core/src/test/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitAdminTestBase.java [hunk 1]
```diff
@@ -723,4 +723,61 @@ public void upgrade_ShouldCallJdbcAdminProperly() throws ExecutionException {
     // Arrange
     verify(distributedStorageAdmin).upgrade(options);
   }
+
+  @Test
+  public void coordinatorNamespaceShouldHandleCorrectly() throws ExecutionException {
+    // Arrange
+    when(admin.getNamespaceNames())
```
Comment (komamitsu) [inline core/src/test/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitAdminTestBase.java:730]:
```suggestion
    when(distributedStorageAdmin.getNamespaceNames())
```
I might be missing something, but it looks the storage admin needs to be used.

**Review by komamitsu — APPROVED** [review]
LGTM! 👍

File: core/src/test/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitAdminTestBase.java (hunk 1 above)
Comment (brfrn169) [reply core/src/test/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitAdminTestBase.java:730]: Ah, that's correct. But why does the current test code work...? Anyway, I will fix it. Thanks.

Comment (brfrn169) [reply core/src/test/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitAdminTestBase.java]: Fixed in c1161422ac6e7deb6460ffd8523ad197fe4b6698. Thank you!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**